celery/canvas.py

```diff
@@ -669,10 +669,20 @@ def prepare_steps(self, args, kwargs, tasks,
                 # signature instead of a group.
                 tasks.pop()
                 results.pop()
-                task = chord(
-                    task, body=prev_task,
-                    task_id=prev_res.task_id, root_id=root_id, app=app,
-                )
+                try:
+                    task = chord(
+                        task, body=prev_task,
+                        task_id=prev_res.task_id, root_id=root_id, app=app,
+                    )
+                except AttributeError:
+                    # A GroupResult does not have a task_id since it is consistent
```

Comment:
I think there's a small typo and it should say "consists" instead of "consistent".